src/vs/workbench/api/common/extHost.api.impl.ts

```diff
@@ -21,15 +22,23 @@ import { ILogService, ILoggerService, LogLevel } from '../../../platform/log/com
 import { getRemoteName } from '../../../platform/remote/common/remoteHosts.js';
 import { TelemetryTrustedValue } from '../../../platform/telemetry/common/telemetryUtils.js';
 import { EditSessionIdentityMatch } from '../../../platform/workspace/common/editSessions.js';
+import { DebugConfigurationProviderTriggerKind } from '../../contrib/debug/common/debug.js';
```

Comment:
Can you ensure to not sort imports in your PRs, I think this adds noise that is not warranted.